lisp/plugins/gst_backend/gst_backend.py

```diff
@@ -120,7 +121,21 @@ def _add_uri_audio_cue(self):
         if files:
             GstBackend.Config["mediaLookupDir"] = os.path.dirname(files[0])
             GstBackend.Config.write()
-
+        self._cue_audio_files(files)
+
+    def cue_audio_urls(self, urls):
+        extensions = self.supported_extensions()
+        files = []
+        for url in urls:
+            filename = url.path()
+            basename, ext = os.path.splitext(url.fileName())
+            if ext.strip('.') not in extensions['audio'] + extensions['video']:
+                # Skip unsupported media types
+                continue
+            files.append(filename)
+        self._cue_audio_files(files)
+
+    def _cue_audio_files(self, files):
```

Comment:
I would name this `add_cue_from_files`, without `_`, it could come useful as "public"